cmd/options/options.go

```diff
@@ -63,6 +69,7 @@ func (npdo *NodeProblemDetectorOptions) AddFlags(fs *pflag.FlagSet) {
 		[]string{}, "List of paths to system log monitor config files, comma separated.")
 	fs.StringSliceVar(&npdo.CustomPluginMonitorConfigPaths, "custom-plugin-monitors",
 		[]string{}, "List of paths to custom plugin monitor config files, comma separated.")
+	fs.BoolVar(&npdo.EnableK8sExporter, "enable-k8s-exporter", true, "Enables reporting to Kubernetes.")
 	fs.StringVar(&npdo.ApiServerOverride, "apiserver-override",
```

Comment:
If k8s exporter is not enabled, this is not necessary. `ValidOrDie()` needs update.

Also, update the description to state that this is ignored if `--enabled-k8s-exporter` is false.

Reply:
Good catch! Done.